lib/req.ex

```diff
@@ -113,7 +113,9 @@ defmodule Req do
 
     * `:url` - the request URL.
 
-    * `:headers` - the request headers. The header names should be downcased.
+    * `:headers` - the request headers as a key-value map or key-value tuple list. 
```

Comment:
strictly speaking they can be any _enumerable_ of `{name, value_or_values}` pairs

i.e. this all works:

```elixir
foo: "bar"

foo: ["bar"]

%{foo: "bar"}

%{foo: ["bar"]}
```

I'm not sure how to best describe it.

Reply:
Ah, I'm not the only one having trouble with that, then 😄 

```suggestion
    * `:headers` - the request headers as a `{key, value}` enumerable (e.g. map, keyword list). 
```

Would this be better?

Reply:
Additional note: given that in request headers it rarely makes sense to pass the same header multiple times, my thought was that it would be OK to omit explicit examples with list-wrapped header values.

Please correct me on this if needed!